experimental/regression_suite/shark-test-suite-models/conftest.py

```diff
@@ -0,0 +1,15 @@
+class VmfbManager:
+    sdxl_clip_cpu_vmfb = None
+    sdxl_vae_cpu_vmfb = None
+    sdxl_unet_cpu_vmfb = None
```

Comment:
Why does this file exist? Can't these just be inlined into the files that use them?

Reply:
We need this file to share data between tests. There is no other way to inline and use the result from one test in another test

Reply:
Which tests share data? I couldn't tell from a high level glance through. A few cases that I looked seemed to only use the vmfb from within the same file, so wouldn't a global in those files serve the same purpose as this separate file?

I want to avoid having a file like this need to know about every configuration of every test.

Reply:
Yeah I tried the global variable method. But, the way it works with those is that you can only consume global variables in all the independent tests. But, we can't change the value in one test for the vmfb and then have the run_module test consume that vmfb generated. The cleanest way I was able to find was this.

(it's just every iree_run_module test uses the vmfb generated by the iree_compile test)